index.html

```diff
@@ -345,6 +345,16 @@ <h2 class="informative">
       which are addressed to ensure the protection of user data during the
       request process.
       </li>
+      <li>Functionality outside the [=user agent=] is out of scope, including
+      platform-specific frameworks, native operating system APIs, standalone
+      applications, and hardware components that store, manage, or process
+      [=digital credentials=]. This includes the user interface and user
+      interactions for selecting a specific credential and obtaining the 
+      user's permission to forward a request to the user-selected 
+      wallet. This specification only defines [=digital credentials=], 
```

Comment:
```suggestion
      wallet. This specification only defines [[digital-credentials]], 
```

This isn't referring to a digital credential itself, it's referring to the API so it shouldn't link to the object.

Reply:
Ok, we should make this more clear, otherwise it's self referencing, which would be confusing. 

We do own "[digital credential](https://w3c-fedid.github.io/digital-credentials/#dfn-digital-credential)" on the W3C side (although we don't currently export it), and the `DigitalCredential` API ("the JavaScript API"). 

Alternatively, we should remove "This specification only defines..." and let the spec speak for itself.